.travis.yml

```diff
@@ -76,18 +76,14 @@ default_test_config: &default_test_config
         - lib32z1-dev
         - gcc-multilib
         - python-dev
-        - python3
         - openssl
         - libssl-dev
   stage: *test
   language: python
   python: &python_version "2.7"
   before_install:
-    # Remove bad openjdk6 from trusty image, so
-    # Pants will pick up oraclejdk6 from `packages` above.
-    - sudo rm -rf /usr/lib/jvm/java-1.6.0-openjdk-amd64
-    - sudo rm -rf /usr/lib/jvm/java-6-openjdk-amd64
-    - jdk_switcher use oraclejdk8
+    - PATH="/usr/lib/jvm/java-8-openjdk-amd64/jre/bin":$PATH
+    - JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64k
```

Comment:
There is a stray `k` on the end here, which makes me wonder whether this `JAVA_HOME` entry is actually being used.

Reply:
That is a very good point. I had removed it, believing it was not needed, but it was causing failures. I'll give it a spin without the `JAVA_HOME`, for sanity's sake.

Reply:
Yeah, absence of `JAVA_HOME` causes a lot of failures.